packages/react-native-bpk-component-switch/src/BpkSwitch.js

```diff
@@ -43,8 +45,11 @@ const getColors = (themeAttributes: ?Object, value: ?boolean): Object => {
   // The color props mean different things based on the platform.
   const colors = Platform.select({
     ios: {
+      ios_backgroundColor: useBpkDynamicValue({
```

Comment:
Can we rename this function to useColors? By convention any function that uses a hook should be name `use` as well.

Reply:
Thanks, updated.